packages/components/nodes/Notion/Notion.ts

```diff
@@ -0,0 +1,581 @@
+              name: 'deleteBlock',
+            },
+          ],
+          default: 'Get Block',
+          show: {
+            'actions.target': ['blockTarget'],
+          },
+        },
+      ] as INodeParams[];
+      this.inputParameters = [
+        {
+          label: 'Notion Version',
+          name: 'notionVersion',
+          type: 'string',
+          optional: false,
+          description: 'Notion Version to be used in the Headers of the request',
+        },
+        {
+          label: 'Page ID',
+          name: 'pageID',
+          type: 'string',
+          optional: false,
+          description: 'ID of the page',
+          show: {
+            'actions.pageOperation': ['getPage', 'updatePage', 'deletePage'],
+          },
+        },
+        {
+          label: 'Database ID',
+          name: 'databaseID',
+          type: 'string',
+          optional: false,
+          description: 'ID of the database',
+          show: {
+            'actions.databaseOperation': [
+              'getDatabase',
+              'updateDatabase',
+              'queryDatabase',
+            ],
+          },
+        },
+        {
+          label: 'Block ID',
+          name: 'blockID',
+          type: 'string',
+          optional: false,
+          description: 'ID of the block',
+          show: {
+            'actions.blockOperation': [
+              'getBlock',
+              'updateBlock',
+              'deleteBlock',
+              'appendBlockChildren',
+              'getBlockChildren',
+            ],
+          },
+        },
+        {
+          label: 'Query String',
+          name: 'queryString',
+          type: 'string',
+          optional: false,
+          description:
+            'Name of the page (with your Notion Integration) to be searched',
+          show: {
+            'actions.target': ['searchTarget'],
+          },
+        },
+        {
+          label: 'Body',
+          name: 'pageBody',
+          type: 'json',
+          description:
+            'Refer to the <a target"_blank" href="https://developers.notion.com/reference/page">Notion API docs</a> on how to structure the request body for page',
+          show: {
+            'actions.pageOperation': ['createPage', 'updatePage'],
+          },
+          placeholder:
+            '"Name": { "title": [ { "text" : { "content": "New page content" } } ] },',
+          optional: false,
+        },
+        {
+          label: 'Body',
+          name: 'databaseBody',
+          type: 'json',
+          description:
+            'Refer to the <a target="_blank" href="https://developers.notion.com/reference/database">Notion API docs</a> on how to structure the request body for database',
+          show: {
+            'actions.databaseOperation': [
+              'createDatabase',
+              'updateDatabase',
+              'queryDatabase',
+            ],
+          },
+          placeholder: '{parent: "parent_Id", properties: "properties"}',
```

Comment:
perhaps a better placeholder too, you can copy what you have been testing that worked correctly or use the example body from https://www.postman.com/notionhq/workspace/notion-s-api-workspace/overview

Reply:
For example this was what worked for me:
```
{
    "parent": {
        "type": "page_id",
        "page_id": "eed256e7-865d-4e12-95bb-c0651e658436"
    },
    "title": [
        {
            "type": "text",
            "text": {
                "content": "Grocery List",
                "link": null
            }
        }
    ],
    "properties": {
        "Name": {
            "title": {}
        }
    }
}
```